### ISSUE TITLE
Staging Review finding: Accessibility: Breadcrumb text clipped when zooming

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#120697]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio-vebt
**Product:** VA Form 22-10297
**Feature:** Application for High Technology Program

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content with errors or inconsistencies.](https://design.va.gov/about/experience-standards/#user-encounters-accurate-consistent-content)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

At zoom levels > 200% and < 400%, breadcrumb text is slightly cut off on the left margin

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="523" height="226" alt="clipped text" src="https://github.com/user-attachments/assets/9877352d-3087-44b2-bb1b-09a0ed8b8367" />

Set browser window at 1280px and using CRL +, zoom in and note breadcrumb display.

screenshot to be added.

</details>

## Recommended action

Ensure breadcrumb margins/placement is consistently on screen. Note: it appears that there are multiple `margin-left` CSS declarations that include `!important`, which may be causing unintended layout issues. 

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** [WCAG 1.4.10 Reflow - Level AA](https://www.w3.org/TR/WCAG22/#reflow)
- **Modality:** Browser zoom
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
